### PR TITLE
unksipped ui test cases

### PR DESCRIPTION
**Pull request by nehajain008**
Story [AB#14286](https://buyingcatalog.visualstudio.com/c5f97979-5b03-4d10-ba8d-871d0526b408/_workitems/edit/14286)
Task [AB#14694](https://buyingcatalog.visualstudio.com/c5f97979-5b03-4d10-ba8d-871d0526b408/_workitems/edit/14694)

### CODE REVIEW

File: app/pages/sections/ordering-party/ui-tests/withoutSavedData.ui.test.js [hunk 1]
```diff
@@ -78,7 +84,8 @@ test('should render organisation ods code with data from OAPI', async (t) => {
     .expect(await extractInnerText(text)).eql(mockDataFromOapi.odsCode);
 });
 
-test('should render organisation address with data from OAPI', async (t) => {
+// TODO: fix failes when all test cases are run
```
Comment (bjss-chris) [inline app/pages/sections/ordering-party/ui-tests/withoutSavedData.ui.test.js:87]: Typo: fails

File: app/pages/sections/ordering-party/ui-tests/withoutSavedData.ui.test.js [hunk 2]
```diff
@@ -106,7 +113,8 @@ test('should render organisation address with data from OAPI', async (t) => {
     .expect(await extractInnerText(addressTextCountry)).eql(mockDataFromOapi.address.country);
 });
 
-test('should render the primary contact details form with populated data from OAPI', async (t) => {
+// TODO: fix failes when all test cases are run
```
Comment (bjss-chris) [inline app/pages/sections/ordering-party/ui-tests/withoutSavedData.ui.test.js:116]: Typo: fails

**Review by bjss-chris — CHANGES_REQUESTED** [review]

File: app/pages/sections/order-items/associated-services/order-item/flat/declarative/ui-tests/withSavedData.ui.test.js [hunk 3]
```diff
@@ -75,10 +79,37 @@ const pageSetup = async (setup = defaultPageSetup) => {
...
+  const quantityInput = Selector('[data-test-id="question-quantity"]');
+  const saveButton = Selector('[data-test-id="save-button"] button');
+
+  await t
+    .typeText(quantityInput, '10', { replace: true })
+    .debug()
+    .click(saveButton)
+    .expect(getLocation()).eql(`http://localhost:1234/order/organisation/${odsCode}/order/${callOffId}/associated-services`);
+});
+
```
[the remark this answers is not in the document]
Comment (nehajain008) [reply app/pages/sections/order-items/associated-services/order-item/flat/declarative/ui-tests/withSavedData.ui.test.js]: done

File: app/pages/sections/order-items/associated-services/order-item/flat/declarative/ui-tests/withSavedData.ui.test.js [hunk 4]
```diff
@@ -75,10 +79,37 @@ const pageSetup = async (setup = defaultPageSetup) => {
...
+  await pageSetup({ ...defaultPageSetup, postRoute: true });
+  await t.navigateTo(pageUrl);
+
+  const quantityInput = Selector('[data-test-id="question-quantity"]');
+  const saveButton = Selector('[data-test-id="save-button"] button');
+
+  await t
+    .typeText(quantityInput, '10', { replace: true })
+    .debug()
```
[the remark this answers is not in the document]
Comment (nehajain008) [reply app/pages/sections/order-items/associated-services/order-item/flat/declarative/ui-tests/withSavedData.ui.test.js]: done

File: app/pages/sections/order-items/associated-services/order-item/flat/declarative/ui-tests/withoutSavedData.ui.test.js [hunk 5]
```diff
@@ -84,12 +85,17 @@ const pageSetup = async (setup = defaultPageSetup) => {
...
+      .reply(200, mockOrgData);
+  })
   .afterEach(async (t) => {
     await nockAndErrorCheck(nock, t);
   });
 
-// TODO: fix when feature completed
-test.skip('should navigate to associated-services dashboard page if save button is clicked and data is valid', async (t) => {
+// TODO: fix when saving associated service issue is resolved
+test('should navigate to associated-services dashboard page if save button is clicked and data is valid', async (t) => {
```
[the remark this answers is not in the document]
Comment (nehajain008) [reply app/pages/sections/order-items/associated-services/order-item/flat/declarative/ui-tests/withoutSavedData.ui.test.js]: done

File: app/pages/sections/order-items/associated-services/order-item/flat/ondemand/ui-tests/withSavedData.ui.test.js [hunk 6]
```diff
@@ -101,6 +106,7 @@ test('should show text fields as errors with error message when there are BE val
   const saveButton = Selector('[data-test-id="save-button"] button');
 
   await t
+    .debug()
```
[the remark this answers is not in the document]
Comment (nehajain008) [reply app/pages/sections/order-items/associated-services/order-item/flat/ondemand/ui-tests/withSavedData.ui.test.js]: done

File: app/pages/sections/ordering-party/ui-tests/withoutSavedData.ui.test.js (hunk 1 above)
Comment (nehajain008) [reply app/pages/sections/ordering-party/ui-tests/withoutSavedData.ui.test.js]: done

File: app/pages/sections/ordering-party/ui-tests/withoutSavedData.ui.test.js (hunk 2 above)
Comment (nehajain008) [reply app/pages/sections/ordering-party/ui-tests/withoutSavedData.ui.test.js]: done

**Review by bjss-chris — APPROVED** [review]

**Pull request merged**